Mathlib/Tactic/Eqns.lean

```diff
@@ -38,8 +38,8 @@ initialize eqnsAttribute : NameMapExtension (Array Name) ←
     add   := fun
     | declName, `(attr| eqns $[$names]*) => do
       if let some _ := Meta.eqnsExt.getState (← getEnv) |>.map.find? declName then
-        throwError "There already exist stored eqns for '{declName}' registering new equations{
-            "\n"}will not have the desired effect."
+        throwError "There already exist stored eqns for '{declName}' registering new equations \
+          will not have the desired effect."
```

Comment:
```suggestion
        throwError "There already exist stored eqns for '{declName}' registering new equations\n\
          will not have the desired effect."
```

Reply:
I slipped in this message change. Having a newline there at that position didn't make sense to me.

@eric-wieser Do you want this newline?

Reply:
@eric-wieser ping

Reply:
No, I agree this newline should be removed

Reply:
Though it looks like there is a mistake in this message:
```suggestion
        throwError "There already exist stored eqns for '{declName}'; registering new equations \
          will not have the desired effect."
```